src/test/moment/is_valid.js

```diff
@@ -122,12 +122,14 @@ test('valid string iso 8601 + timezone', function (assert) {
         '2010-01-30T23:59:59.999+00:00',
         '2010-01-30T23:59:59.999-07:00',
         '2010-01-30T00:00:00.000+07:00',
-        '2010-01-30T00:00:00.000+07'
```

Comment:
Can we just add new test cases not modify existing ones? Now we're not testing 2 digit time zone.